### PR TITLE
treewide: Use formatted logrus logs when possible

### PR DESCRIPTION
**Pull request by pchaigno**
Quick follow up to https://github.com/cilium/cilium/pull/16917#pullrequestreview-779775696.

### CODE REVIEW

File: pkg/bpf/bpffs_migrate.go [hunk 1]
```diff
@@ -170,7 +174,7 @@ func finalizeMap(bpffsPath, name string, revert bool) error {
 	// Pending Map was found on bpffs and needs to be reverted.
 	if revert {
 		dest := filepath.Join(bpffsPath, name)
-		log.Infof("Reverting map pin from '%s' to '%s' after failed migration", file, dest)
+		log.WithField(logfields.BPFMapPath, dest).Infof("Repinning without '%s' suffix after failed migration", bpffsPending)
```
Comment (ti-mo) [inline pkg/bpf/bpffs_migrate.go]: Shouldn't we also include `BPFMapName` since `name` is in scope here?

File: pkg/bpf/bpffs_migrate.go [hunk 2]
```diff
@@ -180,7 +184,7 @@ func finalizeMap(bpffsPath, name string, revert bool) error {
 		return nil
 	}
 
-	log.Infof("Unpinning map '%s' after successful recreation", file)
+	log.WithField(logfields.BPFMapPath, file).Info("Unpinning map after successful recreation")
```
Comment (ti-mo) [inline pkg/bpf/bpffs_migrate.go]: Same as above

**Review by ti-mo — APPROVED** [review]
Neat, thanks! I think we could add `BPFMapName` in a few places, see comments.

**Review by ti-mo — APPROVED** [review]
:shipit:

**Review by joamaki — APPROVED** [review]

**Pull request merged**